libtransmission/rpc-server.c

```diff
@@ -587,21 +606,26 @@ static void handle_request(struct evhttp_request* req, void* arg)
             }
         }
 
-        if (!isAddressAllowed(server, req->remote_host))
-        {
-            send_simple_response(req, 403,
-                "<p>Unauthorized IP Address.</p>"
-                "<p>Either disable the IP address whitelist or add your address to it.</p>"
-                "<p>If you're editing settings.json, see the 'rpc-whitelist' and 'rpc-whitelist-enabled' entries.</p>"
-                "<p>If you're still using ACLs, use a whitelist instead. See the transmission-daemon manpage for details.</p>");
-        }
-        else if (server->isPasswordEnabled && (pass == NULL || user == NULL || strcmp(server->username, user) != 0 ||
+        if (server->isPasswordEnabled && (pass == NULL || user == NULL || strcmp(server->username, user) != 0 ||
             !tr_ssha1_matches(server->password, pass)))
         {
             evhttp_add_header(req->output_headers, "WWW-Authenticate", "Basic realm=\"" MY_REALM "\"");
-            send_simple_response(req, 401, "Unauthorized User");
+            attempts++;
+            char* unauthuser = tr_strdup_printf(
+                "Unauthorized User. "
+                "%i unsuccessful login attempts.",
```

Comment:
It's not a big deal but the rest of the code uses `%d` for `int` formatting, and uniform style is always preferred.